elements/table.js

```diff
@@ -196,12 +194,15 @@ function tableElement (dats, send) {
 // ([obj], fn) -> html
 function createTable (dats, send) {
   return dats.map(dat => {
-    const progress = dat.stats.blocksTotal
-      ? Math.round(dat.stats.blocksProgress / dat.stats.blocksTotal * 100)
+    const stats = dat.stats && dat.stats.get()
```

Comment:
not a fan of getters / setters / proxies as an API b/c it hides what's going on; but yeah think you're right - it's set always now; was being somewhat defensive (: